biogrid/apps/biogrid-mvp/src/app/pages/input-page/input-page.tsx

```diff
@@ -94,88 +96,121 @@ export const InputPage = () => {
-      <h1>Welcome to the Biogrid Simulator!</h1>
-      
-      <p>This website will simulate how a <a href="https://www.sciencedirect.com/science/article/pii/S136403211830128X" target="_blank">microgrid</a> can 
-        optimize energy use in a community by using <a href="https://biomimicry.org/what-is-biomimicry/" target="_blank">biomimicry</a> to 
-        imitate the human body's glucose regulation procedures. </p>
-
-        <p>Hover over the inputs to learn how they're used in the simulation!</p>
-
-        <p>If you would like to learn more about how we designed this project click <a href="" onClick={redirectToInfo}>here</a>.</p>
-          
-        <div className="tooltip">
-          <span className="tooltiptext">The date where we will start collecting sunlight data from</span>
-          <div className="inputBox">
-            <label>Start Date</label>
-            <DatePicker
-              showPopperArrow={false}
-              selected={startDate}
-              onChange={(date: Date) => setStartDate(date)}
-            />
-          </div>
-        </div>
-
-        <div className="tooltip">
-          <span className="tooltiptext">The date where we will stop collecting sunlight data</span>
+        <h1>Welcome to the Biogrid Simulator!</h1>
+
+        <p>
+          This website will simulate how a{' '}
+          <a
+            href="https://www.sciencedirect.com/science/article/pii/S136403211830128X"
+            target="_blank"
+          >
+            microgrid
+          </a>{' '}
+          can optimize energy use in a community by using{' '}
+          <a href="https://biomimicry.org/what-is-biomimicry/" target="_blank">
+            biomimicry
+          </a>{' '}
+          to imitate the human body's glucose regulation procedures.{' '}
+        </p>
+
+        <p>
+          Hover over the inputs to learn how they're used in the simulation!
+        </p>
+
+        <p>
+          If you would like to learn more about how we designed this project
+          click{' '}
+          <a href="" onClick={redirectToInfo}>
+            here
+          </a>
+          .
+        </p>
+
+        <div className="tooltip slider">
+          <span className="tooltiptext">
+            The date where we will start collecting sunlight data from
+          </span>
           <div className="inputBox">
-            <label>End Date</label>
-            <DatePicker
-              showPopperArrow={false}
-              selected={endDate}
-              onChange={(date: Date) => setEndDate(date)}
+            <label>Start Date Slider </label>
+            <ReactSlider
+              className="horizontal-slider"
+              onChange={(val) => {
+                const daysFromStart = (val as unknown) as number;
+                const newDate = new Date();
+                newDate.setDate(EARLIEST_DATE.getDate() + daysFromStart);
+                setStartDate(newDate);
```

Comment:
the way this is rendering now - it looks like a text date and it's not clear at all that it should be dragged, or that dragging it will change the start date, https://screenshot.googleplex.com/9QVY1VAFqHH.png

Reply:
Added a background and a better looking slide